forbnicate.cpp

```diff
@@ -14,22 +14,26 @@ int forbnicate(unsigned int x) {
     x = x<<1; 
```

Comment:
Looks good, just one comment: next time please intend your code before pushing. Most text editors and IDEs support codei intending, but you still can do it online: https://codebeautify.org/cpp-formatter-beautifier

Reply:
ok after doing all this comment ça se passe tawa en général ? genre riguelt kima 9otli est ce que je refais un pull request wala c'est bon je peux directement n7otou ?

Reply:
You do your changes locally and push the commit: this PR gets updated automatically.